File: 2026-05-deblur/example/zig/solution.zig
// Exemplo passthrough: copia stdin pra stdout sem alterar nada.
const std = @import("std");

pub fn main() !void {
    var buf: [8192]u8 = undefined;
    const stdin = std.io.getStdIn().reader();
    const stdout = std.io.getStdOut().writer();
    while (true) {
        const n = try stdin.read(&buf);
        if (n == 0) break;
        try stdout.writeAll(buf[0..n]);
    }
}

File: 2026-05-deblur/example/python/solution.py
#!/usr/bin/env python3
"""Exemplo mínimo: lê o BMP de stdin e devolve sem alterar nada.

Não passa do threshold de PSNR — serve só pra mostrar o contrato de I/O
e como empacotar uma submissão. Copie este diretório, troque a lógica de
`solution.py` pela sua, e mantenha o Dockerfile (ou substitua pela
linguagem/runtime que preferir)."""
import sys

sys.stdout.buffer.write(sys.stdin.buffer.read())
